howso/engine/trainee.py

```diff
@@ -1171,19 +1123,15 @@ def analyze(
                   action_features parameter.
 
         use_case_weights : bool, default False
-            (Optional) When True will scale influence weights by each
+            When True will scale influence weights by each
             case's weight_feature weight.
         use_deviations : bool, default False
             When True, uses deviations for LK metric in queries.
         weight_feature : str, optional
             Name of feature whose values to use as case weights.
             When left unspecified uses the internally managed case weight.
-        kwargs
+        **kwargs
```

Comment:
Python will include all un-specified, named parameters into a mapping called `kwargs`, so, I think it was more correct before.

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html#parameters the numpydoc standard says to leave the stars in place:

```
*args : tuple
    Additional arguments should be passed as keyword arguments
**kwargs : dict, optional
    Extra arguments to `metric`: refer to each metric documentation for a
    list of all possible arguments.
```

We could change this, of course, but this is why the changes were made.